Demo does no run on Windows 10

<!-- Please do not submit support requests or "How to" questions here. For that, go to [StackOverflow](http://stackoverflow.com/questions/tagged/react-admin). -->

**What you were expecting:**
<!-- Describe what the behavior would be without the bug. -->
For the demo app to run in my Windows environment

**What happened instead:**
<!-- Describe how the bug manifests. -->
Same error in this post `data-generator` not found https://github.com/marmelab/react-admin/issues/2030

**Steps to reproduce:**
<!--  Please explain the steps required to duplicate the issue, especially if you are able to provide a sample application. -->
Use Windows 10 Pro version 1809
* Clone the repo `git clone https://github.com/marmelab/react-admin.git`
* `> cd react-admin`
* `make install`
* `make run-demo`

**Environment**

* React-admin version:
* Last version that did not exhibit the issue (if applicable):
* React version:
* Browser:
* Stack trace (in case of a JS error):
```
Failed to compile
./src/fakeServer/graphql.js
Module not found: Can't resolve 'data-generator' in 'C:\Users\david\Documents\react-admin\examples\demo\src\fakeServer'
```

--- a/README.md
+++ b/README.md
@@ -166,8 +166,10 @@ yarn install
 # Run the simple application
 make run-simple
 # Run the tutorial application
+make build
 make run-tutorial
 # Run the demo application
+make build
 make run-demo
 ```
 

--- a/examples/demo/README.md
+++ b/examples/demo/README.md
@@ -17,6 +17,8 @@ After having cloned the react-admin repository, run the following commands at th
 ```sh
 make install
 
+make build
+
 make run-demo
 ```
 

